amethyst_gltf/src/format/mod.rs

```diff
@@ -60,8 +66,10 @@ impl StdError for GltfError {
         match *self {
             GltfImporterError(_) => "Gltf import error",
             InvalidSceneGltf(_) => "Gltf has no default scene, and the number of scenes is not 1",
-            PrimitiveMissingInGfx(_) => "Primitive missing in gfx",
+            //PrimitiveMissingInGfx(_) => "Primitive missing in gfx",
```

Comment:
Remove commented out code please

Reply:
Right, I forgot readding the logic that use that one, good catch.

Reply:
Ah, we only support triangle lists currently, so I'll remove for now and make a note in my todo.